src/main/java/address/controller/RootLayoutController.java

```diff
@@ -54,15 +51,14 @@ public RootLayoutController() {
     }
 
     @FXML
-    private void initialize() {
-        updateSaveLocationDisplay();
-    }
+    private void initialize() {}
```

Comment:
Can't call `updateSaveLocationDisplay` here because the `modelManager` is not passed in yet in `setConnections`. Moved it to `setConnections` instead. Might be better to abstract it out into some sort of initialize method that is outside of FXML initialize.